# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/scnlib/hotfix-cmake.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- scnlib-4.0.1/CMakeLists.txt.vanilla	2026-09-27 17:03:12.420717440 +0300
+++ scnlib-4.0.1/CMakeLists.txt	2026-09-27 17:03:21.187311864 +0300
@@ -88,11 +88,11 @@
 
 add_subdirectory(scripts)
 
-add_subdirectory(benchmark)
+#add_subdirectory(benchmark)
 add_subdirectory(docs)
-add_subdirectory(examples)
+#add_subdirectory(examples)
 
 enable_testing()
-add_subdirectory(tests)
+#add_subdirectory(tests)
 
 include(cmake/install.cmake)
